.github/workflows/helm.yaml

```diff
@@ -39,6 +43,22 @@ jobs:
           if [[ -n "$changed" ]]; then
             echo "::set-output name=changed::true"
           fi
-
+      
       - name: Run chart-testing (lint)
         run: ct lint --config=.github/ci/ct.yaml --validate-maintainers=false
+      
+      # Need a multi node cluster so descheduler runs until evictions
+      - name: Create multi node Kind cluster
+        run: |
+          kind create cluster --name kind --config ./hack/kind_config.yaml --wait 2m
```

Comment:
@knelasevero what step install the `kind` cli? and how can I control the verison?

Reply:
Not sure if kind comes pre-baked in this gh-actions image or if it is installed by one of the previous actions :thinking: